src/biogeophys/FrictionVelocityMod.F90

```diff
@@ -262,12 +262,12 @@ subroutine InitHistory(this, bounds)
             ptr_patch=this%ram1_patch, default='inactive')
     end if
 
-    if (use_cn) then
+    !if (use_cn) then
```

Comment:
This removes the if for only doing this when BGC is one. This should at least have the if removed rather than commented out.

Should we make this happen by default? I could also turn it on by default when use_cn is .true. So could just change when the default is on for use_cn.

Reply:
In talking with @slevis-lmwg we decided we will just make this default off, and have the user add it when it might be helpful for them.

I will also add it explicitly to the Leung dust emission tests.